blockchain/src/webhooks/notifiers/contribution/index.ts

```diff
@@ -42,6 +42,11 @@ export default class ContributionNotifier implements Notifier {
 
     block.tx.forEach(tx => {
       tx.vout.forEach(vout => {
+        // Some vouts are not transactions with addresses, ignore those
```

Comment:
Curious, does this represent coinbase outputs?

Reply:
It was opcode functions, in the case I ran into it was some random `OP_RETURN`

Reply:
For posterity, this is what the JSON looked like:
```
{
  asm: 'OP_RETURN 57616c6c657420697320677265617421',
  hex: '6a1057616c6c657420697320677265617421',
  type: 'nulldata'
}
```